1/p와-y개수-세기.js

```diff
@@ -0,0 +1,22 @@
+// p와 y개수 세기
+
+function solution(s) {
+  let answer = true;
+  let pNum = 0; // p 개수 저장 변수
+  let yNum = 0; // y 개수 저장 변수
+  for (let i of s) {
+    // s 문자열 단어 하나씩 반복
+    if (i === "p" || i === "P") {
+      // p 또는 P면
+      pNum += 1; // p 개수 1 증가
+    } else if (i === "y" || i === "Y") {
+      // y 또는 Y면
+      yNum += 1; // y 개수 1 증가
+    }
+  }
+  if (pNum === yNum) answer = true;
+  // p개수와 y개수가 같으면 answer에 true 저장
+  else answer = false; // 아니면 false 저장
+
+  return answer;
```

Comment:
```suggestion
  if (pNum === yNum) return true;
  else return false;
```
if문에서 조건문 자체가 참이라면 true를 자동으로 반환하기 때문에 answer에 굳이 담아주기보다 바로 값을 return해 주는게 좋을 것 같아요. 
용훈님이 작성하신 코드 같은 경우는 answer가 계속 재할당되어  값이 변경될 때 사용하는게 좋겠네요.

그리고 삼항 연산자를 이용할 수도 있어요!
return (pNum === yNum) ? true : false;